scanners/subfinder/scanner/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# SPDX-FileCopyrightText: the secureCodeBox authors
```

Comment:
A dockerfile for the scanner shouldn't be required as you're using the subfinder image directly :)
You can just delete the scanner folder entirely in that case

Reply:
but you are running the image directly :D
the values.yaml file references the official image.
this might work in tests as it builds up the image locally there but if a user installs the chart later on they get the official image.

Probably the easier way to fix this warning is to specify a user in the helm values under podSecurityContext. 
https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod

See runAsUser and runAsGroup. You can specify ids which don't exist in the image. generally everything over 1000 or better over 10.000 is good :)